test/services/ReviewServiceSpec.scala

```diff
@@ -421,6 +464,73 @@ class ReviewServiceSpec extends BaseSpec with MockFactory with ReviewData with A
       }
     }
 
+    "the process fails to be published due to a duplicate processCode in the published repository" should {
+      "indicate a duplicate key error" in new ReviewCompleteTest {
+
+        val expectedChangeStatusResponse: RequestOutcome[Unit] = Right(())
```

Comment:
I don't think all of the mocking statements are required. Sbt reports that expectedChangeStatusResponse is not used.

Reply:
updated